tests/Filesystem/FilesystemTest.php

```diff
@@ -339,7 +339,7 @@ public function testSharedGet()
             $result *= $status;
         }
 
-        $this->assertTrue($result === 1);
+        $this->assertFalse($result === 1);
```

Comment:
I think only boolean is being returned. If i write `$this->assertTrue($result)` or `$this->assertFalse($result)` tests always returns an error.